packages/components/src/table/index.js

```diff
@@ -128,7 +128,13 @@ class TableCard extends Component {
 	}
 
 	onClickDownload() {
-		const { query, onClickDownload, title } = this.props;
+		const { query, onClickDownload, searchBy, title } = this.props;
+
+		// Delete unnecessary items from filename.
+		delete query.extended_info;
```

Comment:
I'm not sure about directly modifying `query`. I think we should clone it before, either using lodash [`omit`](https://lodash.com/docs/4.17.11#omit) or `Object.assign()`.

Reply:
I'd second this, we typically make copies and don't modify existing objects/arrays.

Reply:
Whoops, thanks for catching this.  Updated.